rules/removing/src/Rector/FuncCall/RemoveFuncCallArgRector.php

```diff
@@ -61,6 +62,10 @@ public function getNodeTypes(): array
     public function refactor(Node $node): ?Node
     {
         foreach ($this->removedFunctionArguments as $removedFunctionArgument) {
+            if ($node->name instanceof MethodCall) {
+                continue;
+            }
```

Comment:
```suggestion
        if ($node->name instanceof Expr) {
            return null;
        }

        foreach ($this->removedFunctionArguments as $removedFunctionArgument) {
```

This avoid dead looping